addons/hw_drivers/tools/helpers.py

```diff
@@ -428,7 +428,8 @@ def list_file_by_os(file_list):
     elif platform_os == 'Windows':
         return [x.name for x in Path(file_list).glob('*[!L].*')]
 
-def odoo_restart(delay):
+
+def odoo_restart(delay=3):
```

Comment:
Add  helper which would be called both from the route and websocket _on_message, something like
```suggestion
def restart_odoo_or_reboot(action, delay=3):
    if action == 'restart_odoo':
        helpers.odoo_restart()
    else:
        helpers.restart_iot_box()
```